custom_components/powerpal/sensor.py

```diff
@@ -94,11 +98,21 @@ def unique_id(self) -> str:
         """Return the unique id."""
         return f"powerpal-live-{self.config_entry.entry_id}"
 
+    @property
+    def device_class(self) -> str:
+        """Return the device class."""
+        return SensorDeviceClass.POWER
+
     @property
     def state_class(self) -> str:
         """Return the state class."""
         return STATE_CLASS_MEASUREMENT
 
+    @property
+    def native_unit_of_measurement(self) -> str:
+        """Return the native unit of measurement."""
+        return UnitOfPower.KILO_WATT
```

Comment:
Should this be `UnitOfEnergy.KILO_WATT_HOUR` the powerpal api returns in watt hours